docs/modules/ROOT/pages/api/governance.adoc

```diff
@@ -667,3 +671,188 @@ Delegates votes from the sender to `delegatee`.
 ==== `[.contract-item-name]#++delegate_by_sig++#++(delegator: ContractAddress, delegatee: ContractAddress, nonce: felt252, expiry: u64, signature: Array<felt252>)++` [.item-kind]#external#
 
 Delegates votes from `delegator` to `delegatee`.
+
+
+[.contract]
+[[VotesComponent]]
+=== `++VotesComponent++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/release-v0.17.0/packages/governance/src/votes/votes.cairo[{github-icon},role=heading-link]
+
+[.hljs-theme-dark]
+```cairo
+use openzeppelin_governance::votes::VotesComponent;
+```
+
+By default, token balance does not account for voting power. This makes transfers cheaper. The downside is that it requires users to delegate to themselves in order to activate checkpoints and have their voting power tracked.
+
+NOTE: When using this module, your contract must implement the {VotingUnitsTrait}. For convinience, this is done automatically for `ERC20` and `ERC721` tokens.
```

Comment:
```suggestion
The Votes component provides a flexible system for tracking and delegating voting power.
It is currently implemented for ERC20 and ERC721 tokens. An account can delegate
their voting power to a representative, that will pool delegated voting units from different
delegators and can then use it to vote in decisions. Voting power must be delegated to be
counted, and an account must delegate to itself if it wishes to vote directly without a trusted
representative.

When integrating the Votes component, the {VotingUnitsTrait} must be implemented to get the
voting units for a given account as a function of the implementing contract. For simplicity,
this module already provides two implementations for ERC20 and ERC721 tokens, which will work
out of the box if the respective components are integrated.

NOTE: ERC20 and ERC721 tokens implementing this component must call {transfer_voting_units}
whenever a transfer, mint, or burn operation is performed. Hooks can be leveraged for this
purpose.
```

Reply:
Why not include the full component description here? Also, splitting into multiple lines as suggested before.

Reply:
All of this is included in the main `Votes` page in `governance.adoc`. Do you suggest I copy/paste what is in there? It feels like unnecessary repetition. 
My expectation is that someone reading the docs looks at the main `Votes` section and then comes to the api for details on function signature, etc. It was ok to have more info in the api section(and I don't think we had much more TBH) when `ERC20Votes` was just an extension.

Reply:
@andrew-fleming , marto and me had this conversation before, and this repetition may be helpful since users in practice don't have to go to the other page first, and documentation should be as easy to understand and follow as possible. Users may come directly to the API page from the READMEs shown in the registry for example, and we shouldn't expect them to go then to the main page and later return here. I vote we add this here too.

Reply:
I understand this... but it also defeats the purpose of having two separate pages and incurs too much repetition. If we were to include things from the main page into api I would say we should only have a single page then.
I was also checking other examples(e.g. finance, [timelock](https://docs.openzeppelin.com/contracts-cairo/0.17.0/api/governance) and even [ERC20](https://docs.openzeppelin.com/contracts-cairo/0.17.0/api/erc20)) and we haven't been following this patterns, so this doesn't seem very consistent. One option that could work is what we did for ERC20 and ad a NOTE with a reference to the main docs.
WDYT?

Reply:
You are right, we don't need a full description here matching what we explained on the main page. Maybe let's rephrase the sentence a bit though, since starting with "By default, token balance does not account for voting power" feels like some context is skipped. A short description like in the other modules, and a couple of notes should suffice.

Reply:
Awesome, added a short description at the beginning of the paragraph